### PR TITLE
docs: add repository intro

### PR DESCRIPTION
**Pull request by Anjellyrika**
📃This PR adds a short introduction to the README file.

### CODE REVIEW

**Comment by mmfallacy** [comment]
LGTM 🐔

**Comment by Dox-Dev** [comment]
LGTM! 🚀

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# 🚀Git Workshop by Basti Ortiz
+
+This repository serves as a check-in for attendees of the **Git Workshop for Modern Software Engineers** hosted by Basti Ortiz, the Director for Engineering of UP CSI.
+
+---
+## Attendees
+
+---
+## Partners
+This workshop is co-hosted by Secuna, in partnership with UP CURSOR and UP ACM.
```
Comment (BastiDood) [inline README.md:10]: Let's add hyperlinks to their websites.
```suggestion
This workshop is co-hosted by [Secuna](https://secuna.io), in partnership with UP CURSOR and UP ACM.
```

**Review by BastiDood — CHANGES_REQUESTED** [review]
Please see the comments in the code.

**Pull request merged**